python/LICENSE

```diff
@@ -0,0 +1,22 @@
+MIT License
```

Comment:
As above: please copy this in from root during build rather than maintaining a duplicate file.